### PR TITLE
Log errors in color yellow

### PR DESCRIPTION
**Pull request by radiumrasheed**
- Added a few more colors 
- log `console.error()` in _yellow_

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -28,7 +28,9 @@ const PORT = config.server.port;
 // Simple function to color text in the console output
 function logColor(msg, color) {
     var colors = {
+		red: ['\x1b[31m', '\x1b[0m'],
```
Comment (MarshallOfSound) [inline server/server.js]: Purely from a pedantic perspective but it looks like the indenting here is a bit off, can you fix that and then I can merge 😄

**Review by MarshallOfSound — CHANGES_REQUESTED** [review]
See comment

File: server/server.js (hunk 1 above)
Comment (radiumrasheed) [reply server/server.js]: oh cool... I live on that perspective too
will fix

**Review by radiumrasheed — COMMENTED** [review]
indentation fixed

**Review by MarshallOfSound — APPROVED** [review]

**Pull request merged**